packages/ui/src/components/editor/panels/Scenes/container/index.tsx

```diff
@@ -56,18 +56,18 @@ export default function ScenesPanel() {
     getMutableState(EditorState).scenePath.set(scene.key)
   }
 
+  useRealtime(fileBrowserPath, scenesQuery.refetch)
```

Comment:
We are already calling useRealtime in FeathersChildReactor for each feathers request. Why do we need to do it again here?

Reply:
It's causing a referch for a different query